src/components/select/select.component.ts

```diff
@@ -381,7 +383,15 @@ export class SelectComponent<T> implements OnInit, OnChanges, OnDestroy, Control
 
     /** This gets called whenever the user types in the input */
     onInputChange(input: string): void {
-        this.inputChange.emit(input);
+
+        this.input = input;
+
+        this._input$.next({
+            value: this.input,
+            userInteraction: true
+        });

```

Comment:
The `this.input` setter emits _input$, so this code looks like it would be emitting the observable twice? Is that expected?

I think this could probably just be this, but I haven't checked it:

```suggestion
        this._input$.next({
            value: input,
            userInteraction: true
        });
```